web/pdf_page_view.js

```diff
@@ -128,7 +128,15 @@ var PDFPageView = (function PDFPageViewClosure() {
         div.removeChild(node);
       }
       div.removeAttribute('data-loaded');
+      div.setAttribute('data-pageremoved-triggered', true);
```

Comment:
I change the following
```javascript
detail: { triggered: true },
```
to something like:
```javascript
detail: { triggered: div.getAttribute('data-page-number')},
```
so the event itself has the page number of the page the event fired on. Then we no longer need the attributes but still have useful information.